docs/rules/html-self-closing.md

```diff
@@ -54,7 +54,8 @@ You can choose either two styles for elements without content
 {
   "svelte/html-self-closing": [
     "error",
-    {
+    "all", // or "html" or "none" (preset)
+    {  // overrides
```

Comment:
I prefer `"all", "html", "none" or object` as the first option rather than splitting the options. I think the override option is confusing for users
What do you think about it?

Reply:
good idea, this would also solve people having to migrate their configurations